packages/common/src/ui/layout/tables/types.ts

```diff
@@ -13,7 +13,7 @@ export enum ColumnFormat {
 }
 
 export interface ColumnDefinition<T> {
-  label: LocaleKey;
+  label?: LocaleKey;
```

Comment:
hm? what did I miss?

Reply:
The checkbox column doesn't need the label